sssd_test_framework/utils/adcli.py

```diff
@@ -0,0 +1,198 @@
+"""Perform actions on Active Directory."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.conn import ProcessResult
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "ADCLIUtils",
+]
+
+
+class ADCLI(MultihostUtility[MultihostHost]):
+    """
+    Call commands from adcli
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        super().__init__(host)
+
+        self.cli: CLIBuilder = self.host.cli
+        """Command line builder."""
+
+        self.fs: LinuxFileSystem = fs
+        """Filesystem utils."""
+
+    def _info(
```

Comment:
Why do you use underscores on the methods name? Those should be all public methods, right?

Reply:
removed